nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -979,23 +994,14 @@ in
         file = ''"$NIX_DISK_IMAGE"'';
         driveExtraOpts.cache = "writeback";
         driveExtraOpts.werror = "report";
+        deviceExtraOpts.bootindex = mkIf cfg.useBootLoader "1";
```

Comment:
Shouldn't this always be bootindex=1 ?

Reply:
I have no a good mental model of how bootindex works in interaction with direct kernel boot. :/
Will investigate.